content/tracing/services/service.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Service
+kind: Documentation
+further_reading:
+- link: "/tracing/services/resource"
+  text: "Dive into your resource performances and traces"
+- link: "/tracing/services/trace"
+  text: "Understand how to read a Datadog Trace"
+---
+
+<div class="alert alert-info"> 
+<a href="https://docs.datadoghq.com/getting_started/apm_tracing">Read our APM getting started page</a>to have a bird-eyed view of all tracing concepts
+</div>
+
+## Overview 
+
+Selecting a service on the services page leads you to the detailed service page:
+
+{{< img src="tracing/services/service/detailed_service_page.png" alt="Detailed service page" responsive="true" popup="true" style="width:90%;">}}
+
+Consult on this page:
+
+* Your [service monitor states](#service-monitor) 
+* Your [out of the box graphs](#out-of-the-box-graphs) 
+* Your [Resources](/tracing/services/resource) 
+
+## Definition
+
+**A service is a set of processes that do the same job.**  
+For instance, a simple web application may consist of two services: 
+
+* A single `webapp` service and a single `database` service
+
+While a more complex environment may break it out into 6 services: 
+
+* 3 separate services: `webapp`, `admin`, and `query`.
+* 3 separate external service:  `master-db`,  `replica-db`, and `yelp-api`.
+
+Explicitly name services by instrumenting your application. For example in Python:  
+
+```python
+from ddtrace import tracer
+# add the `wrap` decorator to trace an entire function
+@tracer.wrap(service='my-app')
+```
+
+Read more about manually instrumenting your application for [Go](/tracing/setup/go/#opentracing-api), [Java](/tracing/setup/java/#manual-instrumentation), [Python](/tracing/setup/python/#example), [Ruby](/tracing/setup/ruby/#example)."
+
+Service names:
+
+* **must be lowercase, alphanumeric characters**.
+* **cannot have more than 50 characters**.
+* **cannot contain spaces** (spaces are replaced with underscores).
+* must adhere to [metric naming rules](/developers/metrics/).
+
+**Note**: Service must have a type attached, [learn more on Service type]().
+
+## Service monitor
+
+Datadog propose list of monitor depending of your service type: 
+
+{{< img src="tracing/services/service/service_monitors.png" alt="Service Monitors" responsive="true" popup="true" style="width:90%;">}}
+
+Enable them directly or create your own [APM monitors](/monitors/monitor_types/apm).
+
+**Note**: Tag any monitor with `service:<SERVICE_NAME>` to attach it to an APM service.
+
+## Out of the box graphs
+
+Datadog provides out of the box graphs for any given Service:
+
+* Requests - Choose to display:
+    *  The **Total amount of requests** 
+    *  The amount of **Requests per second**
+* Latency -  Choose to display:
+    *  The Avg/p75/p90/p95/p99/Max latency of your traced requests 
+    *  The **[Apdex](/tracing/faq/how-to-configure-an-apdex-for-your-traces-with-datadog-apm) score** of your service (for *web* type service only)
```

Comment:
Let's move this page to the Product Specs section - maybe it makes sense to just rename it FAQ or something like that.  It would be helpful if the image indicated that you could see these additional metrics with the dropdowns.  A lot of people don't know we have Apdex, for example, because they don't see it.